### PR TITLE
Global style

### PR DESCRIPTION
**Pull request by willmt80**
## Why

[Monday.com Ticket](https://code4community-team.monday.com/boards/702429655/views/12631042/pulses/829576071)

What benefit does this bring to the end user? Or, what benefit does this bring to developers working in the codebase?

This allows developers to use antd components like text and buttons with the automatic styles that are used throughout the application

## This PR

Describe the changes required and any implementation choices you made to give context to reviewers.

All of the variables are located within the craco.config.js file. The colors are at the top of the page and they correspond to different greens and greys that exist in the figma designs. Any of these values can be overridden in less files, see the info-card.less file for an example of overriding antd global styles.

## Screenshots

Provide screenshots of any new components, styling changes, or pages.

Here is what one of the template page looks like with the new theme. No code or styling was changed on this page, the new colors come from the global style

theme

## Verification Steps

What steps did you take to verify your changes work? These should be clear enough for someone to be able to clone the branch and follow the steps themselves.

Checked the styling on all of the current pages, removed instances of p and h from application to ensure styling works everywhere.

### CODE REVIEW

**Review by florisdobber — COMMENTED** [review]
Looks great to me, super helpful! Let Jack take a look to see if the React makes sense.

**Review by florisdobber — COMMENTED** [review]
Perfect I was looking into doing this but couldn't figure it out

File: craco.config.js [hunk 1]
```diff
@@ -8,7 +8,31 @@ module.exports = {
...
+            modifyVars: {
+              '@text-grey': '#AFAEAE',
+              '@header-grey': '#F5F5F5',
+              '@light-green': '#9AC356',
+              '@mid-green': '#61802E',
+              '@dark-green': '#3A681A',
+              '@black': '#000000',
+              '@white': '#FFFFFF',
+              '@body-background': '@white',
+              '@component-background': '@white',
```
Comment (jackblanc) [inline craco.config.js:20]: Nice job using variables for this

File: src/components/greeting-container/GreetingContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  readonly body: string;
+}
+
+const GreetingContainer: React.FC<GreetingContainerProps> = ({
+  header,
+  body,
+}) => {
+  return (
+    <div className="info-container">
+      <Title style={{ color: '#000000' }}>{header}</Title>
```
Comment (jackblanc) [inline src/components/greeting-container/GreetingContainer.tsx:18]: Is there a way to do this using variables?

File: src/components/greeting-container/greeting-container.less [hunk 3]
```diff
@@ -0,0 +1,19 @@
+.info-container {
```
Comment (jackblanc) [inline src/components/greeting-container/greeting-container.less:1]: We want to start using styled components for things like this, but I don't think it's installed on this repo so we can do it later, after a merge from frontend-scaffold

File: src/containers/signup/signup.less [hunk 4]
```diff
@@ -6,6 +6,11 @@
   width: 100%;
   background: white;
   overflow: auto;
+
+  .ant-typography.footer {
```
Comment (jackblanc) [inline src/containers/signup/signup.less:10]: We want to avoid overwriting AntD styles, why is this change required?

**Review by jackblanc — APPROVED** [review]
Love the progress with this. There are a few things we'll want to change to use https://styled-components.com/ in the future, but we have to merge frontend-scaffold first.

**Review by jackblanc — COMMENTED** [review]
This looks pretty good, there's a few places where we want to use styled-components now. Sorry that the directions changed with that. Reach out on Slack with questions!

**Comment by willmt80** [comment]
Changed some places to use styled components, no longer override antd styles and removed inline magic color constants. Did not change the input container stuff because I believe that might be a separate ticket that involves reorganizing those components a bit more heavily.

File: src/containers/login/login.less [hunk 5]
```diff
@@ -4,19 +4,15 @@
   padding: 8%;
   height: 100%;
   width: 100%;
-  background: white;
+  background: @white;
```
Comment (jackblanc) [inline src/containers/login/login.less:7]: 🔥

**Review by jackblanc — APPROVED** [review]
Looks great!
